testsuite/features/minkvm_guests.feature

```diff
@@ -154,6 +162,13 @@ Feature: Be able to manage KVM virtual machines via the GUI
     And "test-vm2" virtual machine on "kvm-server" should have 1 NIC using "default" network
     And "test-vm2" virtual machine on "kvm-server" should have a "test-vm2_system.qcow2" virtio disk
 
+@virthost_kvm
+  Scenario: Show the Spice display
```

Comment:
"console" :smile_cat:

Reply:
Oops... forgot to change it in the scenario names :wink:

Reply:
fixed